scripts/gen_csr_test.py

```diff
@@ -0,0 +1,220 @@
+"""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Parse processor-specific CSR description YAML file and generate a directed CSR test
+"""
+
+import sys
+import yaml
+import argparse
+import random
+from bitstring import BitArray as bitarray
+# To install the bitstring library:
+#   1) sudo apt-get install python3-bitstring OR
+#   2) pip install bitstring
+
+
+"""
```

Comment:
Please follow the docstring style guide here
http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings